packages/formstr-app/src/index.css

```diff
@@ -60,6 +69,7 @@ code {
 }
 
 .ant-dropdown-menu {
+  width:200px;
```

Comment:
shhould this be a system wide limitation? should this not be on the dropdown we're interested in? i.e. the dashboard dropdown